### PR TITLE
@dzucconi - implements new rule for constructing artist title tags

### PR DESCRIPTION
**Pull request by broskoski**
cc @NicoletteChao @christine-artsy 

See https://trello.com/c/8G5Q0HOl/296-title-tag-ctr-test-artist-pages for more context.

### CODE REVIEW

File: schema/artist/meta.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  const count = artist.published_artworks_count;
+
+  if (count < 10) { return ''; }
+  if (count < 100) { return `${Math.floor(count / 10) * 10}+ `; }
+  if (count < 1000) { return `${Math.floor(count / 100) * 100}+ `; }
+  if (count < 10000) { return `${Math.floor(count / 1000) * 1000}+ `; }
+  return `${Math.floor(count / 10000) * 10000}+ `;
+};
+
+const metaName = (artist) => {
```
Comment (broskoski) [inline schema/artist/meta.js]: This comes from force's implementation of artist title tags. Seems prudent to keep in the weird defensiveness it has.

File: schema/artist/index.js [hunk 2]
```diff
@@ -86,6 +87,7 @@ const ArtistType = new GraphQLObjectType({
       alternate_names: {
         type: new GraphQLList(GraphQLString),
       },
+      meta: Meta,
```
Comment (dzucconi) [inline schema/artist/index.js]: Thankuuu

File: schema/artist/meta.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+};
+
+const ArtistMetaType = new GraphQLObjectType({
+  name: 'ArtistMeta',
+  fields: {
+    title: {
+      type: GraphQLString,
+      resolve: (artist) => {
+        if (slugs.indexOf(artist.id) !== -1) {
+          return `${artist.name} Art - ${countDisplay(artist)}Works, Bio, News | Artsy`;
```
Comment (dzucconi) [inline schema/artist/meta.js]: :\

Comment (broskoski) [inline schema/artist/meta.js]: what would you prefer, sire

**Pull request merged**